docs/contributing.md

```diff
@@ -0,0 +1,75 @@
+# Contributing definitions
+
+Contributing library definitions is as easy as sending a pull request!
+
+All definitions sit under the
+[/definitions](https://github.com/flowtype/flow-typed/tree/master/definitions)
+directory. They all must follow the following directory structure and naming
+format:
+
+```
+└ definitions/npm/
+  ├ yargs_v4.x.x/           # <-- The name of the library, followed by _v<VERSION>
+  | |
+  | ├ flow_v0.23.x/         # <-- A folder containing libdefs tested against the
```

Comment:
Should we bump those examples min version to `0.83.x` which is now the minimum advised version.